cmd/bacalhau/docker_run.go

```diff
@@ -41,7 +42,10 @@ func init() { // nolint:gochecknoinits // Using init in cobra command is idomati
 		&jobInputs, "inputs", "i", []string{},
 		`CIDs to use on the job. Mounts them at '/inputs' in the execution.`,
 	)
-
+	dockerRunCmd.PersistentFlags().StringSliceVarP(
+		&jobInputUrls, "input-urls", "u", []string{},
+		`URL:path of the input data volumes downloaded from a URL source. Mounts data at 'path' (e.g. '-u http://foo.com/bar.tar.gz:/app/bar.tar.gz' mounts 'http://foo.com/bar.tar.gz' at '/app/bar.tar.gz').`, // nolint:lll // Documentation, ok if long.
```

Comment:
what happens if there is a `:` in the URL?

Reply:
That case should work because unlike the other storage providers here I split URL & Path by. the position of the right most `:`

https://github.com/filecoin-project/bacalhau/blob/c8488515e8ff1000356dbb4ebf81d8be54476fc1/pkg/job/job.go#L43

This way you can use a port number in URL:

```
go run . --api-port=$BACALHAU_API_PORT_0 --api-host=localhost docker run \
    --input-urls https://gist.githubusercontent.com:443/enricorotundo/990f0ad01a50d08dfb580e4ad404870e/raw/aa6934257351a0da93f1e740c72f27128590cebc/foo_data.txt:/app/foo_data.txt \
    ubuntu cat /app/foo_data.txt
```

Reply:
I've expanded the description string to clarify that 👍